src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -152,7 +161,7 @@ FillContext buildFillContext(const PROBLEME_HEBDO* problemeHebdo, int NumInterva
                         * nTsInDay;
         lastTimestep = firstTimestep + nTsInDay - 1;
     }
-    return {firstTimestep, lastTimestep};
+    return {firstTimestep, lastTimestep, 0}; // TODO: handle scenarios/year
```

Comment:
```suggestion
    return {firstTimestep, lastTimestep, problemeHebdo->year}; // TODO: handle scenarios/year
```

Reply:
Without this tiny detail, scenarization in hybrid mode won't work

Reply:
Probably, but hybrid is out of scope of the PR